src/server/array_func.c

```diff
@@ -806,15 +806,17 @@ create_subjob(job *parent, char *newjid, int *rc)
  *		PBS_Batch_Rerun, and PBS_Batch_RunJob subjob requests.
  *		For any other request types, be sure to add another switch case below
  *		(matching request type).
+ * @return int
+ * @retval return value of the callback function
```

Comment:
please add return value descriptions in doxygen.
also here you return 0 for failure, but in req_deljob2 0 is success - can we make it the same?